launcher/CMakeLists.txt

```diff
@@ -89,6 +89,10 @@ set(CORE_SOURCES
     # Time
     MMCTime.h
     MMCTime.cpp
+
+    # MangoHud
+    MangoHud.h
+    MangoHud.cpp
```

Comment:
These should only be included on Linux.

Reply:
Agreed.

MangoHud does have some basic support for Windows (https://github.com/flightlessmango/MangoHud/issues/501) but I don't think its usable enough to support.